### DISCUSSION TITLE
examples/http3_server.py : giving error when trying to run the server

### DISCUSSION THREAD

**Discussion by ar-ag** [aiortc/aioquic#508] [Q&A]
I am trying to follow the examples provided under aioquic/examples but , on running the http3 server using the command

``` python3 examples/http3_server.py --certificate tests/ssl_cert.pem --private-key tests/ssl_key.pem```

the following error is appearing

`Traceback (most recent call last):
  File "/home/ar_ag/mininet/aioquic/examples/http3_server.py", line 550, in <module>
    default=defaults.max_datagram_size,
AttributeError: 'QuicConfiguration' object has no attribute 'max_datagram_size'. Did you mean: 'max_datagram_frame_size'?`

**Comment by jlaine** [comment]
I'm not seeing this error, are you sure you have the latest version of aioquic installed? This looks like a version mismatch between `aioquic` and the examples.

**Comment by ar-ag** [comment]
@jlaine Thank you! 
You were right, I had the version 0.9.20 before, I updated that to 1.0.0, and it is working now
